### PR TITLE
[addons] remove the 'void ADDON_Stop()' call

### PR DESCRIPTION
**Pull request by AlwinEsch**
## Description
Due to support of multiple instances must be (if used) the stop be done
on his instance part.

The Stop is currently used mostly only on "Screensaver" and "Visualisaton"
and added there a own "Stop".

This change need on all addons a update!

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp [hunk 1]
```diff
@@ -31,7 +31,6 @@ CInputStreamAddon::CInputStreamAddon(const CFileItem& fileitem, std::shared_ptr<
 CInputStreamAddon::~CInputStreamAddon()
 {
   Close();
-  m_addon->Stop();
```
Comment (ksooo) [inline xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp]: I guess you checked that removing this call has no strange side effects?

**Review by ksooo — APPROVED** [review]
Only one question regarding input stream change, otherwise looking good.

File: xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp (hunk 1 above)
Comment (AlwinEsch) [inline xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp]: No becomes not used on Inputstream addon's. By my look over the addons is the Stop only used on Screensaver and Visualization.

Currently I go over all addons to check. The pull requests to them comes this time after them is in, are to much to make before.

Comment (ksooo) [inline xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp]: Okay , then.

**Comment by AlwinEsch** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h [hunk 2]
```diff
@@ -111,21 +111,21 @@
...
 #define ADDON_INSTANCE_VERSION_VFS_DEPENDS            "kodi_vfs_dll.h" \
                                                       "kodi_vfs_types.h" \
                                                       "kodi_vfs_utils.hpp"
 
-#define ADDON_INSTANCE_VERSION_VISUALIZATION          "1.0.0"
-#define ADDON_INSTANCE_VERSION_VISUALIZATION_MIN      "1.0.0"
+#define ADDON_INSTANCE_VERSION_VISUALIZATION          "1.0.1"
+#define ADDON_INSTANCE_VERSION_VISUALIZATION_MIN      "1.0.1"
 #define ADDON_INSTANCE_VERSION_VISUALIZATION_XML_ID   "kodi.binary.instance.visualization"
 #define ADDON_INSTANCE_VERSION_VISUALIZATION_DEPENDS  "xbmc_vis_dll.h" \
```
Comment (ksooo) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h]: Not sure, but shouldn't the version numbers for the other add-on types changed in this PR (audio encoder etc.) also be bumped?

Comment (ksooo) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h]: As discussed on Slack, please update to include the missing bumps.

**Comment by AlwinEsch** [comment]
jenkins build this please

**Review by ksooo — APPROVED** [review]
Good to go, imo.

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**